apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.html

```diff
@@ -109,6 +109,42 @@
             </div>
         </div>
 
+        <div class="size-selection">
+            <div
+                class="small-box"
+                [ngStyle]="{
+                    'background-color': this.addToBasketForm.get('dimension').value === 'SMALL' ? '#AF7949' : '#FFFFFF'
+                }"
+            >
+                <span class="horizontal" *ngIf="getDesignType() === 'family-tree'">20 cm</span>
+                <span class="horizontal" *ngIf="getDesignType() === 'quotable'">15 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'family-tree'">20 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'quotable'">15 cm</span>
+            </div>
+            <div
+                class="medium-box"
+                [ngStyle]="{
+                    'background-color': this.addToBasketForm.get('dimension').value === 'MEDIUM' ? '#AF7949' : '#FFFFFF'
+                }"
+            >
+                <span class="horizontal" *ngIf="getDesignType() === 'family-tree'">25 cm</span>
+                <span class="horizontal" *ngIf="getDesignType() === 'quotable'">20 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'family-tree'">25 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'quotable'">20 cm</span>
```

Comment:
```suggestion
                <span *ngIf="getDesignType() === 'family-tree'"> 
                   <span class="horizontal" 25 cm</span>
                   <span class="vertical">25 cm</span>
                </span>
                <span *ngIf="getDesignType() === 'quotable'">
                   <span class="horizontal">20 cm</span>
                   <span class="vertical">20 cm</span>
                </span>
```

This is just to make it more clear what is going on in the code. It also saves a little functionality since it only has to run ngIf twice instead of 4 times.